src/annotator/test/integration/hypothesis-injector-test.js

```diff
@@ -65,7 +65,7 @@ describe('HypothesisInjector integration test', () => {
     createHypothesisInjector();
 
     const validFramePromise = new Promise(resolve => {
-      isLoaded(validFrame, () => {
+      isDocumentReady(validFrame, () => {
```

Comment:
Outside the scope of this PR, but these tests would have a more straightforward control flow if `isDocumentReady` returned a promise.

Reply:
I am currently working on this.

Reply:
See https://github.com/hypothesis/client/pull/3670